cmd/vic-machine/configure/configure.go

```diff
@@ -85,6 +105,59 @@ func (c *Configure) Flags() []cli.Flag {
 			Destination: &c.upgrade,
 			Hidden:      true,
 		},
+		cli.BoolFlag{
+			Name:        "regenerate-certs",
+			Usage:       "Generate self-signed certs and enable them on this VCH automatically",
+			Destination: &c.regenerateCerts,
+		},
+		cli.StringFlag{
+			Name:        "tls-key",
+			Value:       "",
+			Usage:       "Virtual Container Host private key file (server certificate)",
+			Destination: &c.skey,
+		},
+		cli.StringFlag{
+			Name:        "tls-cert",
+			Value:       "",
+			Usage:       "Virtual Container Host x509 certificate file (server certificate)",
+			Destination: &c.scert,
+		},
+		cli.StringFlag{
+			Name:        "tls-cname",
+			Value:       "",
+			Usage:       "Common Name to use in generated CA certificate when requiring client certificate authentication",
+			Destination: &c.cname,
+		},
+		cli.StringFlag{
+			Name:        "cert-path",
+			Value:       "",
+			Usage:       "The path to check for existing certificates and in which to save generated certificates. Defaults to './<vch name>/'",
+			Destination: &c.certPath,
+		},
+		cli.BoolFlag{
```

Comment:
this is no tls verify, not no-tls. No TLS is no encryption. No TLS verify means no authentication/authorization against the endpoint, but encryption is enabled.